cmd/geth/misccmd.go

```diff
@@ -103,7 +103,7 @@ func version(ctx *cli.Context) error {
 	}
 	fmt.Println("Architecture:", runtime.GOARCH)
 	fmt.Println("Protocol Versions:", eth.ProtocolVersions)
-	fmt.Println("Network Id:", ctx.GlobalInt(utils.NetworkIdFlag.Name))
+	fmt.Println("Network Id:", eth.DefaultConfig.NetworkId)
```

Comment:
I think we should delete this line altogether. It doesn't make sense to just print `1` here, and noone will do `geth --allmyflags version` anyway. It's imho a mostly useless feature.